pulsar4s-akka-streams/src/main/scala/com/sksamuel/pulsar4s/akka/streams/PulsarCommittableSourceGraphStage.scala

```diff
@@ -26,9 +27,13 @@ trait CommittableMessage[T] {
   def message: ConsumerMessage[T]
 }
 
-class PulsarCommittableSourceGraphStage[T](create: () => Consumer[T], seek: Option[MessageId])
-  extends GraphStageWithMaterializedValue[SourceShape[CommittableMessage[T]], Control]
-    with Logging {
+class PulsarCommittableSourceGraphStage[T](
+  create: () => Consumer[T],
+  seek: Option[MessageId],
+  // If source terminates uncleanly, wait this long for unacked messages before closing consumer
```

Comment:
Can you clarify what you mean by uncleanly? Calling the stop method is "clean" is it not ?

Reply:
Yeah you're right, "cleanly" is not the right term. I was thinking about failures but this also applies to invoking `stop`.

Reply:
I added the explanation in the Scaladoc instead of here